bin/broker-server-start.sh

```diff
@@ -0,0 +1,44 @@
+#!/bin/bash
```

Comment:
@gwenshap Thanks, makes sense. We can have a patch for the original script but keep the script name the same.